Resources/services.xml

```diff
@@ -3,13 +3,7 @@
            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <imports>
-        <import resource="services/commands.xml"/>
-        <import resource="services/components.xml"/>
-        <import resource="services/loggers.xml"/>
-        <import resource="services/managers.xml"/>
-        <import resource="services/subscribers.xml"/>
-        <import resource="services/cronjobs.xml"/>
-        <import resource="services/adyen.xml"/>
-        <import resource="services/validators.xml"/>
+        <import resource="services/*.xml"/>
+        <import resource="services/**/*.xml"/>
```

Comment:
I feel like this is a dangerous change. Better keep the imports explicit.

Reply:
I have reverted the services.xml but I find it less dangerous with the mindset of:
a service configuration that should not be read, should not be present in Resources/services folder.

Currently one file like that exists now: `Resources/services/version/563.xml`.
This file is not included in the `Resources/services/services.xml` so that indicates:
* it should be read, but it is forgotten
* it is no longer used and was forgotten to be removed